### DISCUSSION TITLE
Compatible Python version

### DISCUSSION THREAD

**Discussion by ArijitSinghEDA** [Roche/pyreadstat#232] [Q&A] [answered]
Hello all!

Just wanted to know which Python versions support this library? Thank you.

**Comment by MDS-JAnthony** [comment, marked as answer]
I have used pyreadstat on Python 3.7, 3.8, 3.9, 3.10 and given that the latest update was after the release of 3.11, I imagine it is also stable on 3.11

**Reply by ArijitSinghEDA** [reply]
Thank you for the quick reply!
